chaco/ticks.py

```diff
@@ -242,6 +243,8 @@ def auto_ticks(
     An array of tick mark locations. The first and last tick entries are the
     axis end points.
     """
+    if isnan([data_low, data_high, bound_low, bound_high]).any():
```

Comment:
maybe add a comment here about this being a fix for a regression and a link to the issue.